pkg/operator/e2e/main_test.go

```diff
@@ -352,6 +352,7 @@ func testRuleEvaluatorDeployment(ctx context.Context, t *testContext) {
 				fmt.Sprintf("--export.label.location=%s", location),
 				fmt.Sprintf("--export.label.cluster=%s", cluster),
 				fmt.Sprintf("--query.project-id=%s", projectID),
+				"--export.user-agent=prometheus-engine-export/0.3.3 rule-evaluator/0.3.3 (mode:kubectl)",
```

Comment:
Is it a big deal to change these with each tag? Or should the tests look more like:
```go
	fmt.Sprintf("--export.user-agent=%s/%s %s %s",
		export.ClientName,
		export.Version,
		fmt.Sprintf("rule-evaluator/%s", export.Version),
		"(mode: kubectl)",
	),
```

and

```go
	fmt.Sprintf("--export.user-agent=%s/%s %s %s",
		export.ClientName,
		export.Version,
		fmt.Sprintf("prometheus-collector/%s", operator.CollectorVersion),
		"(mode: kubectl)",
	),
```

Reply:
Yea it's a bit of a hassle I think but I figured the test setup may notably need to change if we do #195 – so I didn't want to optimize this in this PR yet.